frontend/src/components/ResultsMap.js

```diff
@@ -85,9 +89,8 @@ const Wrapper = styled.div`
   position: absolute;
   top: 95px;
   left: 0;
-  //z-index: 5;
 
-  section {
+  div > section {
```

Comment:
I don't see a section in the JSX above, would be more readable with a className or restyling of a styled component.